ruleset/sca/centos/7/cis_centos7_linux.yml

```diff
@@ -1070,7 +1070,7 @@ checks:
       - tsc: ["CC5.2"]
     condition: all
```

Comment:
Please, change this `condition` to `none` to make the check pass when `xorg-x11-server` is not installed.